rocAL/include/loaders/circular_buffer.h

```diff
@@ -54,9 +54,9 @@ class CircularBuffer {
     void unblock_writer();  // Unblocks the thread currently waiting on get_write_buffer
     void push();            // The latest write goes through, effectively adds one element to the buffer
     void pop();             // The oldest write will be erased and overwritten in upcoming writes
-    void set_image_info(const decoded_image_info& info) { _last_image_info = info; }
+    void set_sample_info(const decoded_sample_info& info) { _last_sample_info = info; }
     void set_crop_image_info(const crop_image_info& info) { _last_crop_image_info = info; }
-    decoded_image_info& get_image_info();
+    decoded_sample_info& get_sample_info();
     crop_image_info& get_cropped_image_info();
```

Comment:
@swetha097 same as above "crop_sample_info"? Seems like a few places

Reply:
The crop_image_info is only specific to images - for random bbox crops. Hence did not make it generic